tshirts.c

```diff
@@ -17,6 +17,11 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+    // Boundary value check
+    assert(size(38) == 'S');
+    assert(size(38) == 'M');
+    assert(size(42) == 'M');

```

Comment:
Do you expect size 42 to be both M and L ? Also for size 38, S and M?

Reply:
> Do you expect size 42 to be both M and L ? Also for size 38, S and M?

We don't have requirement about the size chart. So from code analysis I found 38 and 42 are not belongs to any size category. So I wrote test case to check if it belongs to any of these size category. 

Now should 38 be checked against 'S' or 'M'. Similarly for 42, should it be checked against 'M' or 'L'?